apps/desktop/src/components/v3/FileContextMenu.svelte

```diff
@@ -373,6 +386,25 @@
 							}
 						}}
 					/>
+					{#if changes.length === 1}
+						<ContextMenuItem
+							label={showInFolderLabel}
+							onclick={async () => {
+								try {
+									const project = await projectService.fetchProject(projectId);
+									const projectPath = project?.path;
+									if (projectPath) {
+										const absPath = await join(projectPath, changes[0]!.path);
+										await showFileInFolder(absPath);
+									}
+									contextMenu.close();
+								} catch (error) {
+									console.error('Failed to show in file manager:', error);
+									toasts.error('Failed to show in file manager');
+								}
```

Comment:
If we removed this try/catch it would mean the error is handled by the global error handler, which means it would get fed into posthog so we could get some details about how many users this fails for.

Using a toast might be a design choice  though... though I wonder if we could do something that lets it still go through the global error handler so we still get the posthog reports